packages/legend-application-studio/src/components/editor/edit-panel/uml-editor/ClassEditor.tsx

```diff
@@ -206,7 +261,12 @@ const PropertyBasicEditor = observer(
     const visitOwner = (): void => editorStore.openElement(property._OWNER);
 
     return (
-      <div className="property-basic-editor">
+      <div ref={ref} className="property-basic-editor">
```

Comment:
We could do an improvement here, where when you drag the property, the container for it becomes a placeholder with dotted line border, so it looks as if you're lifting the property entry off the page to move it elsewhere.